packages/apps/composer-app/docs/README.md

```diff
@@ -189,13 +187,11 @@ dxos/rpc-tunnel --> dxos/rpc
 dxos/composer-app --> dxos/react-appkit
 dxos/react-client --> dxos/client
 dxos/react-client --> dxos/react-async
-dxos/react-appkit --> dxos/react-uikit
-dxos/react-uikit --> dxos/react-client
-dxos/react-uikit --> dxos/react-ui
+dxos/react-appkit --> @dxos/react-components
 dxos/react-appkit --> dxos/telemetry
 dxos/telemetry --> dxos/sentry
 dxos/composer-app --> dxos/react-composer
-dxos/react-composer --> dxos/react-uikit
+dxos/react-composer --> @dxos/react-componentskit
```

Comment:
I know this is generated but where did "componentskit" come from?

Reply:
Oh, it's not generated, I may have made a goof. Will fix.